modules/preview-react/form-field/lib/FormFieldContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import {createSubcomponent} from '@workday/canvas-kit-react/common';
+import {handleCsProp, createStyles, CSProps} from '@workday/canvas-kit-styling';
+
+import {useFormFieldModel} from './hooks';
+
+const containerBaseStyles = createStyles({
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+export const FormFieldContainer = createSubcomponent('div')({
+  displayName: 'FormField.Container',
+  modelHook: useFormFieldModel,
+})<CSProps>((elemProps, Element) => {
+  return <Element {...handleCsProp(elemProps, [containerBaseStyles])} />;
+});
```

Comment:
Helper container to align the hint text when the position of the label is horizontal